include/zephyr/shell/shell.h

```diff
@@ -1241,6 +1241,15 @@ int shell_obscure_set(const struct shell *shell, bool obscure);
  */
 int shell_mode_delete_set(const struct shell *shell, bool val);
 
+/**
+ * @brief Retrieve return value of most recently executed shell command.
+ * 
+ * @param[in] shell Pointer to the shell instance
+ * 
```

Comment:
trailing space.

This commit is missing signoff and body: `Shell: add retrieve return value via shell API`